pages/features.js

```diff
@@ -30,7 +27,7 @@ export default function FeaturesPage() {
           <Feature>
             <Icon>
               <img
-                src="/static/img/features/icons/git-icon.svg"
+                src="/img/features/icons/git-icon.svg"
```

Comment:
same, don't see a good reason to change URLs here

Reply:
All this stuff back in /static/... now! Thanks